### PR TITLE
Filebeat upgrade to v7.8.1

### PR DESCRIPTION
**Pull request by rafzei**
This probably should be merged after 0.7.1 release

### CODE REVIEW

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/templates/filebeat.yml.j2 [hunk 1]
```diff
@@ -256,17 +301,30 @@ processors:
...
+# output configuration, so if you have the Elasticsearch output configured such
+# that it is pointing to your Elasticsearch monitoring cluster, you can simply
+# uncomment the following line.
+#monitoring.elasticsearch:
+
+# ================================= Migration ==================================
+
+# Enable the compatibility layer for Elastic Common Schema (ECS) fields.
+# This allows to enable 6 > 7 migration aliases.
+#migration.6_to_7.enabled: true
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/filebeat/templates/filebeat.yml.j2:330]: \n

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt [hunk 2]
```diff
@@ -36,7 +36,7 @@ elasticsearch-oss-6.8.5
 elasticsearch-oss-7.3.2 # Open Distro for Elasticsearch
 erlang-21.3.8.7
 ethtool
-filebeat-6.8.5 # actually it's filebeat-oss
+filebeat-7.8.1 # actually it's filebeat-oss
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt:39]: is this comment still valid? I do not understand it. Maybe add link to some place explaining it or delete it.

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/requirements.txt [hunk 3]
```diff
@@ -35,7 +35,7 @@ elasticsearch-oss-6.8.5
 elasticsearch-oss-7.3.2 # Open Distro for Elasticsearch
 erlang-21.3.8.7
 ethtool
-filebeat-6.8.5 # actually it's filebeat-oss
+filebeat-7.8.1 # actually it's filebeat-oss
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/redhat-7/requirements.txt:38]: same here

File: core/src/epicli/data/common/ansible/playbooks/roles/filebeat/defaults/main.yml [hunk 4]
```diff
@@ -1,3 +1,3 @@
 ---
 specification:
-  filebeat_version: "6.8.5"
\ No newline at end of file
+  filebeat_version: "7.8.1"
```
Comment (mkyc) [inline core/src/epicli/data/common/ansible/playbooks/roles/filebeat/defaults/main.yml:3]: \n

File: core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt (hunk 2 above)
Comment (rafzei) [reply core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt:39]: Not sure what is the purpose of this comment. I can remove it.

Comment (rpudlowski93) [reply core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt:39]: I understand the change but maybe in order to clarify the lines should we change the name filebeat-7.8.1 to filebeat-oss-7.8.1? The same as in elasticsearch case. It is important to now which version we use (opensource code). It's just my proposal

Comment (rafzei) [reply core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt:39]: All Beats are oss so I don't think it's needed.

Comment (rpudlowski93) [reply core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt:39]: Nice to know! Thanks:)

**Pull request closed without merging**